### DISCUSSION TITLE
Sloeber V4.4 Linux - Trying to compile debug configuration

### DISCUSSION THREAD

**Discussion by doughaliburton55** [Sloeber/arduino-eclipse-plugin#1464] [Q&A]
I have followed the video for Sloeber V4.2 esp8266 debug configuration. I am able to compile the Release version of a sketch without errors. When I change the configuration to a debug build and change the platform folder to jantje/hardware/esp8266/1.0.0, the compile fails. It appears that there are some include files missing. One path that is missing is the one for the sdk/include. 
This path is in the release configuration, but not in the debug configuration. Something to do with the platform folder being different I assume. 
What is the best way to compile a debug version? If the sdk/include path must be added, how is this done? 
It looks like it should work in the video. I followed along but maybe something has changed from V4.2 to V4.4?

**Comment by jantje** [comment]
That video is old and the esp8266 platform version it worked on is old as well.
But with the options to add parameters at the end of the compile commands you should be able to compile a debug version without anything special

FYI The reason this works is because gcc compile options later in the command line overwrite the previous ones.
